bin/run_web_disk_manager.sh

```diff
@@ -17,7 +17,7 @@ cd /app/
 export PROCESS_NAME=disk_manager
 
 SLEEP_SECONDS=60
-PROCESS_TIMEOUT_SECONDS=120
+PROCESS_TIMEOUT_SECONDS=240
```

Comment:
This feels weird to me. If it takes more than 2 minutes to run, I wonder if the problem is less that it doesn't have time to finish and more that it's hanging somewhere.

Reply:
I fixed the `remove_orphaned_files` timing graph and then looked at the last 7 days and it does get up to 2 minutes. I double-checked the code and there's no obvious places it would hang. If it's hanging, it's in the python vm or function implementations.

I'm game for increasing this to 240.